_inc/client/searchable-modules/index.jsx

```diff
@@ -0,0 +1,135 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Component } from 'react';
+import { connect } from 'react-redux';
+import forEach from 'lodash/forEach';
+import includes from 'lodash/includes';
+import { translate as __ } from 'i18n-calypso';
+import Banner from 'components/banner';
+
+/**
+ * Internal dependencies
+ */
+import { getModules, isModuleActivated, activateModule } from 'state/modules';
+import { isModuleFound } from 'state/search';
+import SettingsCard from 'components/settings-card';
+import SettingsGroup from 'components/settings-group';
+import { userCanManageModules } from 'state/initial-state';
+import { isDevMode, isUnavailableInDevMode } from 'state/connection';
+
+class SearchableModules extends Component {
+	render() {
+		// Only admins plz
+		if ( ! this.props.canManageModules ) {
+			return null;
+		}
+
+		// Only render if search terms present
+		const searchTerms = this.props.searchTerm;
+		if ( searchTerms.length < 3 ) {
+			return null;
+		}
+
+		// Only should be features that don't already have a UI, and we want to reveal in search.
+		const whitelist = [
+			'contact-form',
+			'custom-css',
+			'enhanced-distribution',
+			'json-api',
+			'latex',
+			'notes',
```

Comment:
Should Monitor be on that list as well, as folks can activate it when it's been deactivated?